WordPress/Classes/Services/AccountService+Swift.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+import WordPressShared
+import ShareExtensionCore
+
+extension AccountService {
+    func setupAppExtensions() {
+        let context = coreDataStack.mainContext
+        context.performAndWait {
```

Comment:
It's not ideal that we block the thread. It's called from the background too, which is especially problematic. I'll probably update it to use a non-blocking call?